src/app/shared/services/logging.service.ts

```diff
@@ -1,15 +1,99 @@
+import {Injectable} from "@angular/core";
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {environment} from "../../../environments/environment";
+import UserProfileInterface from "../../_interfaces/userProfile.interface";
+
+@Injectable({
+  providedIn: 'root'
+})
 export class LoggingService {
   private isLogged: boolean = false;
 
+  constructor(private http: HttpClient) {}
+
   isAuthenticated() {
-    return this.isLogged;
+    const time = localStorage.getItem('authTokenLastUntil');
+    return new Date().getTime() < Number(time);
+  }
+
+  register(email: string, password: string, location: string) {
```

Comment:
There should be return type in every method => register(email: string, password: string, location: string): Observable<{}> as an example. Don't use empty object